package.json

```
@@ -25,10 +25,10 @@
     "README.md",
     "package.json"
   ],
-  "packageManager": "pnpm@8.0.0",
+  "packageManager": "pnpm@9.0.1",
```

perhaps go for latest while at it?
```suggestion
  "packageManager": "pnpm@9.1.4",
```